### PR TITLE
Configuring for zope-product

### PR DESCRIPTION
**Pull request by icemac**
Update to current meta/config version.

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -20,6 +20,7 @@ skip_install = true
 deps =
     setuptools < 52
     zc.buildout
+    zope.testrunner
```
Comment (jugmac00) [inline tox.ini:23]: Is this necessary?

Comment (icemac) [reply tox.ini]: https://github.com/zopefoundation/Products.ZMySQLDA/pull/26/commits/88c89f6863ec09389a9834bb34c654005f605b42 tries without that dependency.

Comment (icemac) [reply tox.ini]: This strangely seems to fix the `py27-zope2` env.

**Review by dataflake — APPROVED** [review]

**Comment by icemac** [comment]
Thank you for reviewing this PR. 😃

**Pull request merged**